packages/contentful--create-contentful-app/src/template.ts

```diff
@@ -22,41 +22,44 @@ async function getGithubFolderNames(username: string, repo: string, path: string
   const response = await fetch(url);
   const contents: any = await response.json();
 
-  return contents.filter((content: any) => content.type === "dir")
-  .map((content: any) => content.name);
+  return contents
+    .filter((content: any) => content.type === 'dir')
+    .map((content: any) => content.name);
 }
-  
-  
+
 async function promptExampleSelection(): Promise<string> {
-  let template = 'typescript'
+  let template = 'typescript';
 
   // ask user whether to start with a blank template or use an example template
   const { starter } = await inquirer.prompt([
     {
       name: 'starter',
-      message: 'Do you want to start with an example template or use one of the blank templates?',
+      message: 'Do you want to start with an example or use one of our blank templates?',
```

Comment:
Now it feels like there are multiple blank templates and only one example. Maybe something like this?
> Do you want to start with a blank template or use one of our examples?

Reply:
And this also aligns nicely with choices:
1. blank
2. example